docs/compose.md

```diff
@@ -37,10 +37,27 @@ functionality that you don't require.
 Currently the customization is limited as described below but future
 releases will increase the amount of customization that is available.
 It is also possible to [build Triton](build.md#building-triton)
-yourself to get more exact customization.
+from source to get more exact customization.
 
-The two Docker images used for customization are retrieved using the
-following commands.
+## Use the compose.py script
+
+The `compose.py` script can be found in the [server repository](https://github.com/triton-inference-server/server). Simply clone the repository and run `compose.py` to create a custom container. 
```

Comment:
Need to help them know which branch to clone. They need to clone the release branch corresponding to the NGC release that they want to base their custom image on. For example they need to use r21.06 branch if they want to create an image based on the NGC 21.06 Triton release.